CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Main
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* None.
+
+##### Bug Fixes
+
+* Yams will now correctly error when it tries to decode a mapping with duplicate keys.  
```

Comment:
This is technically a bug fix but it may be breaking if people have malformed yaml they are trying to decode with this